desktop/frontend/src/pages/api/pgsql/getAll.ts

```diff
@@ -0,0 +1,31 @@
+import * as k8s from '@kubernetes/client-node';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { CRDMeta, K8sApi, ListCRD, GetUserDefaultNameSpace } from 'services/backend/kubernetes';
+import { JsonResp } from '../response';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { kubeconfig } = req.body;
+  const kc = K8sApi(kubeconfig);
+  const kube_user = kc.getCurrentUser();
+
+  if (kube_user === null) {
```

Comment:
We have common resp status at file:  `src/pages/api/response.ts` to return frontend same error message struture.